homeassistant/components/shopping_list.py

```diff
@@ -34,36 +35,47 @@
 SERVICE_COMPLETE_ITEM = 'complete_item'
 
 SERVICE_ITEM_SCHEMA = vol.Schema({
-    vol.Required(ATTR_NAME): vol.Any(None, cv.string)
+    vol.Required(ATTR_NAME): vol.Any(None, cv.string),
+    vol.Optional(ATTR_LIST_ID, '0'): cv.string
```

Comment:
I don't think that we should have our IDs be just numbers or have `'0'` be the default one. I think that it really depends on what lists are available. Would just not give it a default value.

Reply:
Default of '0' is so the existing panel will still work and voice intents and I know what it is.

Reply:
Is the idea that future lists will be 1, 2, 3 etc ? And if I remove 2, will there be a hole or will all subsequent IDs switch one back?

I think that I would prefer the following approach:

- Convert the current list to a system generated list named by default "Inbox" that the user cannot delete. The ID will be hardcoded to `system-inbox`. 

The ids of all other lists will be generated using `uuid.uuid4().hex`

Reply:
No, my plan is actually exactly the approach you stated. I should have made that clear, besides the id being `system-inbox`, I like that change.

Reply:
@balloob do you think I should have a way to allow current shopping-list users to move their current items or is clearing them okay if listed as a breaking change?